vertx-web-api-contract/src/main/java/io/vertx/ext/web/api/contract/RouterFactoryOptions.java

```diff
@@ -237,4 +245,19 @@ public RouterFactoryOptions addGlobalHandler(Handler<RoutingContext> globalHandl
     this.globalHandlers.add(globalHandler);
     return this;
   }
+
+  public String getExposeConfigurationKey() {
+    return exposeConfigurationKey;
+  }
+
+  /**
```

Comment:
Please explain in the doc that another handler is added to route handlers and if this parameter is null no handler will be added